src/TMxpEntityTagHandler.cpp

```diff
@@ -52,7 +53,17 @@ TMxpTagHandlerResult TMxpEntityTagHandler::handleStartTag(TMxpContext& ctx, TMxp
         if (tag->hasAttribute("PUBLISH")) {
             client.publishEntity(entity, value);
         }
-    }
+    } else if (!tag->hasAttribute("ADD") && !tag->hasAttribute("REMOVE")) {
+        // Apparently there is no (or an empty) value, so set us to empty string.
+        // Adding or removing an empty string to the value has no effect, that's why we skipped those.
+        // Note that the TagParser will parse a '' argument as a non existent argument, so
+        // so both <!EN name> and <!EN name ''> will come here.
+        const QString& value = "";
```

Comment:
Qt objects get default intialised, so you can just do this:

```suggestion
        const QString value;
```